pom.xml

```diff
@@ -755,9 +723,13 @@ limitations under the License.
           <useDefaultExcludes>false</useDefaultExcludes>
 
           <includes>
-            <include>src/**</include>
-            <include>*</include>
-            <include>**/.mvn/**</include>
+            <include>*.xml</include>
```

Comment:
why is this plugin configuration changed?

Reply:
TBH, I don't recall. I *think* I ran into some differences wrt copyright headers at that time, but that's obviously no longer the case, so I reverted this change.